src/main/bin/ccm-fetch

```diff
@@ -299,10 +302,22 @@ if (!defined($errno)) {
 } elsif ($errno == $ERROR) {
     $this_app->error("Unable to fetch profile");
     exit(1);
-} elsif ($errno == CHANGED) {
-    $this_app->info("Profile updated");
 } else {
-    $this_app->verbose("Fetched profile was not new");
+    # Init cachemanager
+    my $cmgr = EDG::WP4::CCM::CacheManager->new($fetch->{CACHE_ROOT}, $fetch->{_CCFG});
+    # get Configuration instance (anonymous, latest CID)
+    my $cfg = $cmgr->getConfiguration();
+    # get name and CID
+    my $msg = "latest CID ".$cfg->getConfigurationId();
+
+    my $name = $cfg->getName();
+    $msg .= " name $name" if defined($name);
```

Comment:
i see not much more value in using `branch` rather than `name`. i used name because it's the result of the `getName` method on the configuration instance (which directlty relates to a profile).

the code changes should be made in the TT file, rather than here.
the current name is actually a TT file, so we can change this to print whatever you want (we can ship more than one, and we can decide what the default is/should be; it's configurable anyway). so it's flexible in that direction. 
we can also add a `human_readable.tt` to print something more human readable (an extreme case would be adding a `motd.tt`. so that's also another direction we can modify the message printed here.

an analogy would be user names: the name is the username (short, but not too abstract), the commit-id the uid (very abstract), and the the humanreadble string is the gecos. (the analogy breaks down on the uniqueness of the name, which is not mandatory in this case).
maybe you'd prefer to see the gecos here, or another naming convention; well, we can all make it configurable.

but the main point of the "name" is something related to the discussion we had wrt atomicity: i would like to introduce some identifier for a profile (and thus also the configuration instance(s)) based on (hopefully only) the `/metadata` field alone (maybe the current one need some more fields), that can be tracked during the life of a profile. 
ideally such an identifier is compact enough, yet contains enough relevance. eg the latest commit id on itself is unique-ish and can be traced back (well, ignoring git history rewrites), but i believe it's a bit too abstract. (i think dave coined the term "profile blame", but it's convenient to have e.g. the author of the branch around. also if you are debugging a broken node, it's nice to have a general idea that the latest profile is the one you expected, whithout having to do some git magic first).

this identifier is what i call the name here. it will pop up in logfiles, e.g. ccm-fetch logs it, when ncm-ncd starts, it could also print it. even the notification to cdp-listend could have it, and passed to ccm-fetch (e.g. `ccm-fetch --expected some_name`, and when another profile is retrieved because someone else is also deploying, it can at least report this, saying, `hey, you expected some_name, but i retrieve some_other_name`; and then you can have a chat with whoever is also deploying this host).